.gitignore

```diff
@@ -36,7 +39,7 @@ cli/test-apps/**
 .DS_Store
 
 # Ignore VSCode settings
-.vscode/
+vscode/
```

Comment:
```suggestion
.vscode/
```
not sure why this changed? it's almost always `.vscode`

Reply:
also not really sure...